model/train.py

```diff
@@ -71,12 +76,13 @@ def main():
         images,
         batch_size=config["batch_size"],
         epochs=config["epochs"],
-        validation_split=config["validation_split"],
+        # validation_split=config["validation_split"],  # FIXME: Does autoencoder need validation data split?
```

Comment:
In our case since likely not: https://stats.stackexchange.com/questions/329248/in-it-necessary-to-split-train-test-validation-dataset-for-unsupervised-machin